src/renderers/webgl/WebGLState.js

```diff
@@ -917,3 +933,40 @@ THREE.WebGLStencilBuffer = function ( gl, state ) {
 	};
 
 };
+
+THREE.WebGLColorMask = function ( r, g, b, a ) {
+
+	this.r = r || true;
+	this.g = g || true;
+	this.b = b || true;
+	this.a = a || true;
+
+	this.set = function ( r, g, b, a ) {
+
+		this.r = r;
+		this.g = g;
+		this.b = b;
+		this.a = a;
+
```

Comment:
It would be great if you can use the constructor without parameters. So if you just call `new THREE.WebGLColorMask()`, all color components flags are set to true. This version would support four parameters, a single one and none.

```javascript
this.r = r || true;
this.g = g || this.r;
this.b = b || this.r;
this.a = a || this.r;
```